tests/baselines/reference/importTag15(module=es2015).js

```diff
@@ -21,4 +20,6 @@ export interface I {
 /** @import { I } from './0' with { type: "json" } */
 /** @import * as foo from './0' with { type: "json" } */
 /** @param {I} a */
-declare function f(a: import("./0").I): void;
+declare function f(a: I): void;
+import { I } from './0' with { type: "json" };

```

Comment:
...We maybe want to transform to `import type`, since that's how we treated the tag?

Reply:
oke